features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/dbscripts/oracle.sql

```diff
@@ -1544,6 +1544,55 @@ CREATE TABLE IDN_CONFIG_FILE (
 )
 /
 
+CREATE TABLE IDN_REMOTE_FETCH_CONFIG(
```

Comment:
Formatting check other table creation as well

Reply:
Fixed in https://github.com/wso2/carbon-identity-framework/pull/3143